test/new-e2e/tests/otel/otel-agent/config/sampling.yml

```diff
@@ -17,7 +17,7 @@ exporters:
 processors:
   batch:
   probabilistic_sampler:
-    sampling_percentage: 50
+    sampling_percentage: 100
```

Comment:
Are really testing sampling if we sample at `100%`?

Reply:
Thanks for catching this, reverted this change.